src/components/copy-page-dropdown.tsx

```diff
@@ -0,0 +1,230 @@
+  title = "Documentation Page",
+}) => {
+  const [copied, setCopied] = useState(false);
+  const [isOpen, setIsOpen] = useState(false);
+  const [markdownUrl, setMarkdownUrl] = useState<string | null>(null);
+  const [hasMounted, setHasMounted] = useState(false);
+
+  // Prevent hydration mismatch
+  useEffect(() => {
+    setHasMounted(true);
+  }, []);
+
+  useEffect(() => {
+    if (copied) {
+      const timeout = setTimeout(() => setCopied(false), 4000);
+      return () => clearTimeout(timeout);
+    }
+  }, [copied]);
+
+  const getCleanHtmlContent = (): HTMLElement | null => {
+    const element = document.getElementById("doc-content");
+    if (!element) {
+      alert("Unable to locate content for export.");
+      return null;
+    }
+
+    const clone = element.cloneNode(true) as HTMLElement;
+    const elementsToRemove = clone.querySelectorAll("[data-exclude-from-md]");
+    for (const el of elementsToRemove) {
+      el.remove();
+    }
+    return clone;
+  };
+
+  const convertToMarkdown = (html: string): string => {
+    const rawMd = htmlToMd(html);
+    return rawMd.replace(/<button>Copy<\/button>\n?/g, "");
+  };
+
+  const handleCopyPage = () => {
+    const htmlContent = getCleanHtmlContent()?.innerHTML || "";
+    const markdown = convertToMarkdown(htmlContent);
+    const referenceSection =
+      "\n\n---\nAsk questions about this page:\n- [Open in ChatGPT](https://chat.openai.com/chat)\n- [Open in Claude](https://claude.ai/)";
+    const finalContent = `${title}\n\n${markdown}${referenceSection}`;
+
+    copy(finalContent);
+    setCopied(true);
+  };
+
+  const exportMarkdownAndOpen = async (): Promise<string | null> => {
+    if (markdownUrl) return markdownUrl;
+
+    const htmlContent = getCleanHtmlContent()?.innerHTML || "";
+    const markdown = convertToMarkdown(htmlContent);
+    const pathname = window.location.pathname.replace(/^\//, "") || "index";
+    const filename = `${pathname}.md`;
+
+    try {
+      const res = await fetch("/api/export-md", {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({ content: markdown, filename }),
+      });
+
+      if (!res.ok) throw new Error("Export failed");
+
+      const data = await res.json();
+      const fullUrl = `${window.location.origin}${data.url}`;
+      setMarkdownUrl(fullUrl);
+      return fullUrl;
+    } catch (err) {
+      alert("Failed to export Markdown.");
+      return null;
+    }
+  };
+
+  const handleViewMarkdown = async () => {
+    const url = await exportMarkdownAndOpen();
+    if (url) window.open(url, "_blank");
+  };
+
+  const openInLLM = async (generateUrl: (url: string) => string) => {
+    const url = await exportMarkdownAndOpen();
+    if (url) window.open(generateUrl(url), "_blank", "noopener,noreferrer");
+  };
+
+  if (!hasMounted) return null;
+
+  return (
+    <div
+      className="inline-flex rounded-lg overflow-hidden lg:mb-0 bg-white item-center w-fit ml-auto"
```

Comment:
```suggestion
      className="inline-flex rounded-lg overflow-hidden lg:mb-0 brand-glass-gradient text-neutral-text shadow-sm item-center w-fit ml-auto"
```

So that we have the colouring configurable for dark mode too